packages/app/src/providers/mock/mock-api.ts

```diff
@@ -115,6 +116,26 @@ export class MockApi extends DataProviderApi {
     return undefined;
   }
 
+  public async getSearchablePaths(path: string): Promise<string[]> {
+    return this.getEntityPaths(path);
+  }
+
+  private getEntityPaths(entityPath: string): string[] {
+    const entity = findMockEntity(entityPath);
+    if (!entity) {
+      return [];
+    }
+
+    if (!isGroup(entity)) {
+      return [entity.path];
+    }
+
+    return entity.children.reduce<string[]>(
+      (acc, child) => [...acc, ...this.getEntityPaths(child.path)],
+      [entity.path]
+    );
+  }
```

Comment:
Implementation of `getSearchablePaths` for the mock API as asked. This was also used for the Cypress tests.

Reply:
Very nice :clap: